include/CppUTest/TestFailure.h

```diff
@@ -135,6 +135,22 @@ class UnsignedLongsEqualFailure : public TestFailure
     UnsignedLongsEqualFailure(UtestShell* test, const char* fileName, int lineNumber, unsigned long expected, unsigned long actual, const SimpleString& text);
 };
 
+#ifdef CPPUTEST_USE_LONG_LONG
+
+class LongLongsEqualFailure : public TestFailure
+{
+public:
+    LongLongsEqualFailure(UtestShell* test, const char* fileName, int lineNumber, long long expected, long long actual, const SimpleString& text);
+};
+
+class UnsignedLongLongsEqualFailure : public TestFailure
+{
+public:
+    UnsignedLongLongsEqualFailure(UtestShell* test, const char* fileName, int lineNumber, unsigned long long expected, unsigned long long actual, const SimpleString& text);
+};
+
+#endif /* CPPUTEST_USE_LONG_LONG */
```

Comment:
I prefer including comments on an `#endif` when sufficiently far away from the corresponding `#if` -- that way the context is obvious even if your editor does not provide block outlines for these.

But if you really hate them, I'm open to stripping them out.